build.sbt

```diff
@@ -67,20 +70,16 @@ val okHttpVersion = "3.12.1"
 
 val bbcBuildProcess: Boolean = System.getenv().asScala.get("BUILD_ORG").contains("bbc")
 
-val bbcCommonLibSettings: SettingsDefinition = if (bbcBuildProcess) {
-  Seq(
-    libraryDependencies ++= Seq("com.gu" %% "pan-domain-auth-play_2-6" % "0.9.2-SNAPSHOT")
```

Comment:
Has using `0.9.2` caused any problems for you?

Reply:
No, but our new authentication provider doesn't require pan-domain-auth any more, so there's no need to have a bbc specific version of it

Reply:
Just earmarking it as a potentially easy target for us to update.